### PR TITLE
cmd/tuf: Remove term package usage and import

### PR DESCRIPTION
**Pull request by titanous**
This import is way more trouble than it's worth, so just remove the usage, which is very minor.

### CODE REVIEW

File: cmd/tuf/main.go [hunk 1]
```diff
@@ -127,13 +126,6 @@ func getPassphrase(role string, confirm bool) ([]byte, error) {
 		return []byte(pass), nil
 	}
 
-	state, err := term.SaveState(0)
-	if err != nil {
-		return nil, err
-	}
-	term.DisableEcho(0, state)
-	defer term.RestoreTerminal(0, state)
-
```
Comment (lmars) [inline cmd/tuf/main.go]: Could switch to the `golang.org/x/crypto/ssh/terminal` package that has a [`ReadPassword`](https://godoc.org/golang.org/x/crypto/ssh/terminal#ReadPassword) function?

Comment (lmars) [reply cmd/tuf/main.go]: Or the seemingly more convenient [`Terminal.ReadPassword`](https://godoc.org/golang.org/x/crypto/ssh/terminal#Terminal.ReadPassword) that includes the prompt too.

Comment (titanous) [reply cmd/tuf/main.go]: Oh, nice. Will do.

**Comment by titanous** [comment]
PTAL

**Pull request merged**